### DISCUSSION TITLE
invalid player data

### DISCUSSION THREAD

**Discussion by twosevensix276** [iPortalTeam/ImmersivePortalsMod#1660] [Non-Compatibility Issue Report]
### Minecraft version

1.21.1

### Describe the issue

everytime i try to play single player i cant play it because it keeps showing "invalid player data" then sending me to the multiplayer screen. is there a fix for this?
![2024-11-25](https://github.com/user-attachments/assets/a89eb4ba-acae-41c2-908a-9a83fdc4def8)

**Comment by Poopooracoocoo** [comment]
Make sure you've updated Immersive Portals to the latest version for your MC version. 6.0.5 fixed that issue.
